internal/genci/githubimpl.go

```diff
@@ -133,13 +135,23 @@ func convertTaskToStep(node *scheduler.Stage) *schema.GithubStep {
 		if gh.If != "" {
 			step.If = gh.If
 		}
+		// if env is specified on this level we want to overwrite it
+		// with ci_meta.github.env keys
+		if gh.Env != nil {
+			step.Env = node.Env().
+				Merge(node.Task.Env).
+				Merge(variables.FromMap(utils.ConvertToMapOfStrings(gh.Env))).
+				Map()
```

Comment:
What does this do without any args?

Reply:
Oh, I now see the `.` on the previous line...

Reply:
ah yes sorry - wasn't sure what you meant - but yes chaining over multiple lines needs the `.` at the end of the line. TS for example it doesn't matter or can be in the beginning of line as well

Reply:
@dnitsch That's a terrible lint pattern XD